src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -261,6 +261,22 @@ const getAuthenticatorData = function(authenticator, isVerifyAuthenticator) {
   return authenticatorData;
 };
 
+const getIDProofingData = function(remediations) {
```

Comment:
Is the idea that this method will eventually include additional data like the methods above? eg. dataSeAttr, AriaLabel, description, etc? 

If this will only return a class name, I would scope this method (name and return content) to just return the class name.

Reply:
Yes, that is the plan. this method can be extensible similar to `getAuthenticatorData`